src/etc/inc/interfaces.inc

```diff
@@ -4424,6 +4430,13 @@ function get_interfaces_info()
             $ifinfo['ipaddr'] = $ifinfo['ipv4'][0]['ipaddr'];
             $ifinfo['subnet'] = $ifinfo['ipv4'][0]['subnetbits'];
         }
+        if (is_dslite_enabled($ifdescr)) {
+            $gifif = dslite_find_tunnel_interface($ifdescr);
+            $aftr = dslite_find_aftr_address($ifdescr);
+            $ifinfo['ipaddr'] = 'dslite';
```

Comment:
the tunnel does not have a local ipv4 tunnel address?

Reply:
It is always 192.0.0.2 -> 192.0.0.1 as per spec, therefore the ip doesn't really make sense to be shown. You share a public IPv4 with multiple customers using the AFTR CNATing.